src/main/java/Practice.java

```diff
@@ -1,60 +1,75 @@
+import java.util.Arrays;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 
 public class Practice {
     public int sum_of_integers(List<Integer> numbers) {
-        throw new UnsupportedOperationException();
+        return numbers.stream().reduce(0, (a, b) -> a + b);
     }
 
     public int sum_of_list_of_lists(List<List<Integer>> numbers) {
-        throw new UnsupportedOperationException();
+        return numbers.stream().map(this::sum_of_integers).reduce(0, (a, b) -> a + b);
     }
 
     public int sum_of_integers_from_one_to_hundred() {
-        throw new UnsupportedOperationException();
+        return IntStream.range(1, 101).reduce(0, (left, right) -> left + right);
```

Comment:
what if I tell you that there is a method for this: `.reduce(0, (left, right) -> left + right)`